packages/react-storage/src/components/FileUploader/ui/FileList/types.ts

```diff
@@ -0,0 +1,59 @@
+import { StorageManagerDisplayTextDefault, TaskHandler } from '../../utils';
```

Comment:
Looks like we missed renaming `StorageManagerDisplayTextDefault`?

Reply:
Updated to use name referencing FileUploader